README.md

```diff
@@ -45,17 +45,24 @@ commands.
 
 ## Parallel Tests
 
-The test reporter supports aggregating multiple test coverage reports from, for example, parallel runs
-together into a single report. After aggregation, the test reporter can send up the single aggregated
-report to Code Climate.
+Support for parallel tests is made possible by using sub-commands provided by the test
+reporter.  Specifically, the test reporter has sub-commands to:
 
-This requires you store the partial payloads yourself after each test, then
-download them to one location before using the reporter to upload a summed
-payload.
+1. format partial results (`format-coverage`)
+2. aggregate partial results into a single result (`sum-coverage`) and
+3. upload the single result to Code Climate (`upload-coverage`)
+
+To make use of these commands, parallel test support requires:
+
+1. the ability to run commands after *each* individual worker has completed (most CI
+systems support this)
+2. the ability to run commands after *all* tests have completed (most CI systems support this)
+2. uploading and downloading partial test coverage data to/from shared storage
+(using AWS S3, for example)
 
 For example:
 
-1. After *each* test:
+1. After *each* worker run:
```

Comment:
This might be a super small detail but circle ci uses the term container for _worker_. And i've used semaphore ci and they also use the term container or box.